site/ic_data_repo/templates/dart_id.html

```diff
@@ -0,0 +1 @@
+<dt class="ui tiny header">{{field_value}}</dt>
```

Comment:
Currently the way this renders on the record detail page isn't very clear:
![image](https://github.com/user-attachments/assets/75bf317f-141e-4e7b-9579-b88ed03ff7ab)

Perhaps:
```suggestion
<dt class="ui tiny header">DART ID - {{field_value}}</dt>
```